src/Serilog.Sinks.RollingFile/Sinks/RollingFile/TemplatedPathRoller.cs

```diff
@@ -29,18 +29,42 @@ class TemplatedPathRoller
         const string OldStyleDateSpecifier = "{0}";
         const string DateSpecifier = "{Date}";
         const string DateFormat = "yyyyMMdd";
+        const string HourSpecifier = "{Hour}";
+        const string HourFormat = "yyyyMMddHH";
+        const string HalfHourSpecifier = "{HalfHour}";
+        const string HalfHourFormat = "yyyyMMddHHmm";
         const string DefaultSeparator = "-";
 
+        const string MatcherMarkSpecifier = "date";
+        const string MatcherMarkInc = "inc";
+
         readonly string _pathTemplate;
         readonly Regex _filenameMatcher;
+        readonly SpecifierTypeEnum _specifierType = SpecifierTypeEnum.None;
+        // Concret used Date or Hour specifier.
```

Comment:
Typo in this comment ("concret") - we could possibly drop the comment since the field names are pretty descriptive.